client-app/src/common/plugins/store.info.plugin.ts

```diff
@@ -0,0 +1,17 @@
+import _Vue from "vue";
```

Comment:
This is not component, of course, but I think we should follow this naming convention any way:
https://vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended
It's ok to treat `.service` or `.plugin` as sub-extension, but file name itself should be kebab-case: `store-info.plugn.ts`

Reply:
That`s right.